packages/features/auth/lib/next-auth-options.ts

```diff
@@ -455,6 +457,7 @@ export const AUTH_OPTIONS: AuthOptions = {
           role: token.role as UserPermissionRole,
           impersonatedByUID: token.impersonatedByUID as number,
           belongsToActiveTeam: token?.belongsToActiveTeam as boolean,
+          organizationId: token?.organizationId,
```

Comment:
Making sure we have the orgId in the session to then rely on it for branding provider